src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -89,6 +90,12 @@ static void DetectCiphersuiteConfiguration()
     {
         // There's no system_default configuration, so no default CipherString.
         ERR_clear_error();
+
+        if (API_EXISTS(SSL_state))
+        {
+            // OpenSSL 1.0 does not support CipherSuites so there is no way for caller to override default
+            g_config_specified_ciphersuites = 1;
+        }
     }
     else
     {
```

Comment:
I can't put a comment far enough down, but if the intent is to just use the builtin default on OpenSSL 1.0, replace the complex `#elif !defined(FEATURE_DISTRO_AGNOSTIC_SSL) && defined(SSL_SYSTEM_DEFAULT_CIPHER_LIST)` with just `#else`